checker/env.go

```diff
@@ -236,6 +241,13 @@ func (e *Env) addIdent(decl *exprpb.Decl) errorMsg {
 	return ""
 }
 
+func (e *Env) isOverloadDisabled(overloadID string) bool {
+	if _, found := e.filteredOverloadIDs[overloadID]; found {
```

Comment:
Just return found and get rid of the if?

Reply:
Done.